dask_bigquery/core.py

```diff
@@ -39,6 +41,26 @@ def bigquery_clients(project_id):
         bq_storage_client.transport.grpc_channel.close()
 
 
+@contextmanager
+def bigquery_client(project_id, credentials=None):
+    """Create the BugQuery Client"""
+    client_info = rest_client_info.ClientInfo(
+        user_agent=f"dask-bigquery/{dask_bigquery.__version__}"
+    )
+    with bigquery.Client(
+        project_id, credentials=credentials, client_info=client_info
+    ) as client:
+        yield client
+
+
+def gcs_client(project_id, credentials=None):
```

Comment:
it looks like you added `gcsfs` to the requirements but are using `google-cloud-storage` here. I would probably vote for using `gcsfs` since that's what `to_parquet` is going to use already, and operations like `glob` are a bit simpler

Reply:
I replaced `google.cloud.storage.Client` with `gcsfs.GCSFileSystem`, but it doesn't make much difference, at least not yet. The reason I added `gcsfs` to reqs is that `to_parquet` call fails without it.